package.json

```diff
@@ -52,6 +54,7 @@
     "@types/file-saver": "^2.0.1",
     "@types/hapi__joi": "^17.1.0",
     "@types/jest": "^24.0.0",
+    "@types/mocha": "file:./src/types/empty-type",
```

Comment:
hmm any reason for this?

Reply:
Noted the comment u added on the gsn imported type, but does the package for @types/mocha break?

Reply:
we don't use @types/mocha for our test so no problem right?

Reply:
nah just thinking if its more maintable if we just install it normally, afterall its dev dep..

Reply:
Err I just realised we need this to override @types/mocha from opengsn/gsn/node_modules so we dont have conflicting types from testcafe and mocha!

https://github.com/microsoft/TypeScript/issues/18588#issuecomment-652019495

Reply:
-.-"

Can open an issue for this on this repo, thanks? So that we know the reasons behind it and we can track it in the future

Reply:
For linking: https://github.com/TradeTrust/document-creator-website/issues/85